include/core/ib_particle.h

```diff
@@ -28,33 +28,49 @@ template <int dim>
 class IBParticle
 {
 public:
+
+
+
+// This class defined values related to a particle used for in the sharp interface immersed boundary.
+    // Each particle defined will have these value used in the solver
+    void initialise_all();
+    void initialise_last();
```

Comment:
Fix indentation
There are some spelling mistakes. Check spelling.
```suggestion
    // This class defines values related to a particle used for in the sharp interface immersed boundary.
    // Each particle defined will have these value used in the solver
    void initialise_all();
    void initialise_last();
```

Last line of comment is not clear.